doc/example/getting_started.cpp

```diff
@@ -112,18 +167,49 @@ MOCK_BASE_CLASS( mock_view, view )
 
 class calculator
 {
+    view& v;
 public:
-    calculator( view& v );
+    calculator( view& v ): v(v) {}
+
+    void add( int a, int b ){ v.display(a + b); }
+};
+
+struct CatchFailureFixture
```

Comment:
Why do we need a custom error policy?

Reply:
To test for failure. We want the test to pass when the mock reports a test failure. In boost test there is no way to check if the current test failed and reset it to pass afterwards so this as a workaround

Reply:
Is this ok or any open question? If not please resolve so I don't loose track of open issues ;)